### ISSUE TITLE
coverity scannning

### ISSUE THREAD

**Issue by jameshilliard** [mchehab/zbar#86]
It might be a good idea to enable coverity scanning on this codebase which is free for open source projects. See [here](https://scan.coverity.com/) for details on signing up. I suspect it would find a number of issues on a large older parser heavy c codebase like this.

**Comment by mchehab** [comment]
Yeah, enabling coverity is interesting (and I have already an account there, with I use for Linux Kernel).

I don't remember anymore what's needed in order to add a new project to it and to parse the sources in order to run the static analyzer.
